wpilibj/src/main/native/cpp/HALUtil.cpp

```diff
@@ -208,17 +209,30 @@ jobject CreatePWMConfigDataResult(JNIEnv *env, int32_t maxPwm,
                         minPwm);
 }
 
-void SetCanStatusObject(JNIEnv *env, jobject canStatus, 
+void SetCanStatusObject(JNIEnv *env, jobject canStatus,
                         float percentBusUtilization,
-                        uint32_t busOffCount, uint32_t txFullCount, 
-                        uint32_t receiveErrorCount, 
+                        uint32_t busOffCount, uint32_t txFullCount,
+                        uint32_t receiveErrorCount,
                         uint32_t transmitErrorCount) {
-    static jmethodID func = env->GetMethodID(canStatusCls, "setStatus", 
-                                             "(DIIII)V");
-    env->CallObjectMethod(canStatus, func, (jdouble)percentBusUtilization, 
-                          (jint)busOffCount, (jint)txFullCount, 
-                          (jint)receiveErrorCount, (jint)transmitErrorCount);
-  }
+  static jmethodID func = env->GetMethodID(canStatusCls, "setStatus",
+                                           "(DIIII)V");
+  env->CallObjectMethod(canStatus, func, (jdouble)percentBusUtilization,
+                        (jint)busOffCount, (jint)txFullCount,
+                        (jint)receiveErrorCount, (jint)transmitErrorCount);
+}
+
+void SetMatchInfoObject(JNIEnv* env, jobject matchStatus,
+                        const HAL_MatchInfo* matchInfo) {
```

Comment:
Actually, even better would be to make this `const HAL_MatchInfo&`.

Reply:
Fixed